src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -101,6 +102,11 @@ tags:
       - `next_cursor` *(optional)* if present indicates that the next chunk (page) exists and can be fetched using collection-specific endpoint and cursor value,
       - `items` a chunk (page) of unerlying collection items.
   - name: Concepts
+  - name: Examples
+    x-displayName: Examples
+    x-traitTag: true
+    description: |
+        TODO FILL IT IN.
```

Comment:
move entire examples.md here so it'll be automatically rendered in redocly as other secitons.